pkg/client/client.go

```diff
@@ -183,6 +184,42 @@ func (cl *Client) UpsertKubernetesSecret(namespace, id, key string, data []byte)
 	return nil
 }
 
+// UpsertKubernetesConfigMap updates the value of key of the kubernetes configMap specified by namespace, name.
+// It inserts a new configMap if id doesn't already exist.
+// Returns nil if successful, error otherwise
+func (cl *Client) UpsertKubernetesConfigMap(namespace, name, key, data string) error {
```

Comment:
It doesn't look like the controller uses this, seems to be test-only, maybe move this over to `fixture.ClientInterface`?

Reply:
I thought it would be a helpful utility function for updating the ConfigMap?
But true that the controller is not using this, maybe I'll first move this to `fixture.ClientInterface`.